test/suites/zombie_tanssi_relay_eth_bridge/test_zombie_tanssi_relay_eth_bridge.ts

```diff
@@ -213,6 +242,26 @@ describeSuite({
                 ).stdout
             );
 
+            const ethereumNetwork = { Ethereum: { chainId: TESTNET_ETHEREUM_NETWORK_ID } };
+
+            wETHAssetId = 42;
```

Comment:
Maybe we can put it to const with a proper name? I saw we use it in several places in tests.